orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -274,6 +284,29 @@ at::Tensor view(const at::Tensor& self, at::IntArrayRef size) {
     self.options());
 }
 
+at::Tensor _cat(at::TensorList tensors, int64_t dim) {
```

Comment:
it seems the only reason our opgen script can't handle it is because the tensor list. have a try to fix that. if it really hard to support, please leave a TODO here, about enhance the opgen script later to support tensor list.

Reply:
Done.